### ISSUE TITLE
add manual random seed for dataset preshuffle

### ISSUE THREAD

**Issue by CunliangGeng** [DeepRank/deeprank#48]
add `preshuffle_seed` option to NeuralNet.py. By giving a fixed seed value, it will generate same training/validation/test data for different trainings, which is useful for optimising hyperparameters, e.g. batch size.
